src/Components/NavBar/NavItem.tsx

```diff
@@ -15,8 +15,11 @@ interface NavItemProps extends BoxProps {
   className?: string
   href?: string
   onClick?: () => void
+  isFullScreenDropDown?: boolean
 }
 
+type Position = React.CSSProperties["position"]
```

Comment:
Can also use styled-systems's `PositionProps` interface here.

Reply:
I tried pull `PositionProps` but was getting the same type error.